applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -137,10 +163,17 @@ static void stream_stopped_cb(struct bt_audio_stream *stream)
 
 	LOG_INF("Audio Stream %p stopped", (void *)stream);
 
-	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
-	ERR_CHK(ret);
+	if (stream->conn == headset_conn[AUDIO_CHANNEL_LEFT]) {
```

Comment:
I think a helper function can be used here. Get index based on stream, Graham created one for BIS stereo as well, but haven't pushed it yet. Something like:
````C
static int get_stream_index(struct bt_audio_stream *stream, uint8_t *index)
{
    for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
        if (&streams[i] == stream) {
            *index = i;
            return 0;
        }
    }

    LOG_WRN("Stream not found");

    return -EINVAL;
}

Reply:
And a similar for conn perhaps